book/src/concepts/system.md

```diff
@@ -134,6 +134,41 @@ This system will make all entities with both a `Transform` with a positive y coo
 
 Cool! Now that looks like something we'll actually do in our games!
 
+### Getting entities that have some components, but not others
+
+There is a special type of `Storage` in specs called `AntiStorage`.
+You can invoke it using the "!" symbol when borrowing a normal `Storage`.
```

Comment:
how about,

```
You can invoke it using the `!` operator when borrowing a normal `Storage`.
```

Reply:
Tbh, invoke sounds a bit odd here. Maybe something like "the not operator (`!`) turns a `Storage` into its `AntiStorage`"?